AODN/MHL/process_MHLsst_from_txt.py

```diff
@@ -247,7 +248,14 @@ def parse_nc_attribute(input_netcdf_file_path, output_nc_obj):
     input: text file *.TXT or *_new.txt
 
     """
+    parser = argparse.ArgumentParser()
 
-    for txtfile in glob.glob('/vagrant/tmp/MHL/TXTFILES/*_new.txt'):
+    parser.add_argument('dir_path', help="Full path to input text file directory")
+    parser.add_argument('txt_file', help="input text file")
+
+    args = parser.parse_args()
+    dir_path = args.dir_path
+    txt_file = args.txt_file
+    for txtfile in glob.glob(os.path.join(dir_path,txt_file)):
```

Comment:
That works, though if you're just going to join `dir_path` and `txt_file` together anyway, why not just have a single input arg for the full path to the file (including glob pattern)?